irdata/xmldata.py

```diff
@@ -94,5 +94,8 @@ def characters(self, content):
         else:
             self.current_chars[current_path] += content
 
+    def handleElement(self, content):
```

Comment:
This makes me a little nervous. Are you confident this is correct (should overload `super`)?

Reply:
@lducazu ping for comment

Reply:
The `handleElement()` method here is just a placeholder. The method is not present in any of the base classes, but it _is_ called in `EmptyElementParser.endElement()`. This is an error and pylint is complaining about it. The reason why the error hasn't surfaced till now is because class `EmptyElementParser` is used in only place: in `irdata_xml2tab.py` (irefindex), where a subclass is created (`PSIParser`) which does provide method `handleElement`.

The same applies to `filenames` in class `Writer` (data.py). `self.filenames` is used in its methods, but the class doesn't provide this -- it needs to be provided by derived classes, eg `PSIWriter` in `irdata_xml2tab.py` (irefindex).

Reply:
Ok perfect. Thanks @lducazu